lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -46,21 +47,24 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      */
     constructor(configuration: Configuration) {
         super(configuration);
-        this.initializeBrokering();
     }
 
     /**
      * 
      */
-    private initializeBrokering(): void {
-        if (this.config.system.brokerOptions.actAsBroker) {
+    async initializeBrokering(): Promise<void> {     
+        if (this.config.system.brokerOptions.actAsBroker && !BrowserUtils.isInIframe()) {
+            if(this.config.system.brokerOptions.allowBrokering) {
+                this.logger.verbose("Broker options actAsBroker and allowBrokering both set to true. Preferring to act as broker.");
```

Comment:
What does "preferring to act as broker" mean?

Reply:
It just means that there are conflicting config options so we are silently respecting the `actAsBroker` flag and ignoring the `allowBrokering` flag. I suppose wording it simply `Acting as broker.` would get the point across as well.

Reply:
Does this imply it is either or and never both? If so can we spell it out?

Reply:
Updated for clarity.